### DISCUSSION TITLE
Statistics about StreetComplete usage

### DISCUSSION THREAD

**Discussion by piebro** [streetcomplete/StreetComplete#4658] [General]
Hey, I gathered some statistics about StreetComplete from the OSM changeset file because I was curious about the impact of StreetComplete that I want to share. Among others, this issue: https://github.com/streetcomplete/StreetComplete/issues/1749 inspired me to start the project.

Once I started, I realized that I could gather many more interesting statics about OSM in general and I created a little website visualizing everything and make the code easily expandable. Here are some plots about StreetComplete I think are interesting. For more interactive statics about StreetComplete and OSM in general, you can check out the website at: https://piebro.github.io/openstreetmap-statistics/ and the code at: https://github.com/piebro/openstreetmap-statistics. Also the data at the repo and website should always be up-to-date since it can easily be updated with GitHub actions.

![edits_made_with_streetcomplete_per_month](https://user-images.githubusercontent.com/17904272/203800428-e0d266f2-403a-42a3-84ba-a05d4257cf7f.png)

There are on average over 1M edits each month in the last two years after an interesting spike at the end of 2020. I wonder why the growth is so sudden.

![percent_of_contributors_that_use_streetcomplete_per_month](https://user-images.githubusercontent.com/17904272/203800411-cb1bcd77-c913-4605-950b-ce326b6f4d17.png)

In the last month, 14.5% of all users contributing to OSM in that month also used StreetComplete to contribute at least once. I think that's quite a big number considering that the app is an independent project and only available for android.

![total_edits_with_streetcomplete_map](https://user-images.githubusercontent.com/17904272/203801088-0f9bc123-68fa-4019-9c4e-b1f7304e13a0.png)

The map shows that Europe is the hotspot for StreetComplete, but it is used all over the world.

<h3>yearly edit count per quest</h3><table style="margin: 0 auto;"><tbody><tr><th>Rank</th><th>StreetComplete</th><th>2016</th><th>2017</th><th>2018</th><th>2019</th><th>2020</th><th>2021</th><th>2022</th><th>Total</th></tr><tr><td>1</td><td>AddRoadSurface</td><td>0</td><td>699,705</td><td>534,706</td><td>428,071</td><td>496,913</td><td>1,270,021</td><td>1,131,025</td><td>4,560,441</td></tr><tr><td>2</td><td>AddBuildingType</td><td>0</td><td>0</td><td>216,587</td><td>277,087</td><td>630,296</td><td>1,736,024</td><td>1,246,019</td><td>4,106,013</td></tr><tr><td>3</td><td>AddWayLit</td><td>0</td><td>108,264</td><td>226,736</td><td>300,578</td><td>485,481</td><td>1,255,455</td><td>789,408</td><td>3,165,922</td></tr><tr><td>4</td><td>AddBuildingLevels</td><td>0</td><td>84,234</td><td>137,679</td><td>159,553</td><td>396,895</td><td>1,191,709</td><td>806,285</td><td>2,776,355</td></tr><tr><td>5</td><td>AddPathSurface</td><td>0</td><td>0</td><td>42,597</td><td>147,386</td><td>305,752</td><td>946,911</td><td>681,030</td><td>2,123,676</td></tr><tr><td>6</td><td>AddRoofShape</td><td>0</td><td>20,328</td><td>53,298</td><td>64,678</td><td>175,129</td><td>859,902</td><td>364,441</td><td>1,537,776</td></tr><tr><td>7</td><td>AddHousenumber</td><td>0</td><td>22,274</td><td>41,388</td><td>46,711</td><td>140,976</td><td>450,658</td><td>328,786</td><td>1,030,793</td></tr><tr><td>8</td><td>AddCycleway</td><td>0</td><td>33,375</td><td>69,445</td><td>41,524</td><td>103,614</td><td>421,130</td><td>350,469</td><td>1,019,557</td></tr><tr><td>9</td><td>AddSidewalk</td><td>0</td><td>0</td><td>0</td><td>33,321</td><td>75,975</td><td>472,282</td><td>344,234</td><td>925,812</td></tr><tr><td>10</td><td>AddLanes</td><td>0</td><td>30</td><td>4</td><td>0</td><td>86,714</td><td>360,932</td><td>264,008</td><td>711,688</td></tr><tr><td>11</td><td>AddTactilePavingCrosswalk</td><td>0</td><td>24,245</td><td>32,007</td><td>40,078</td><td>69,995</td><td>247,549</td><td>214,068</td><td>627,942</td></tr><tr><td>12</td><td>AddAddressStreet</td><td>0</td><td>0</td><td>2</td><td>0</td><td>82,162</td><td>272,991</td><td>254,076</td><td>609,231</td></tr><tr><td>13</td><td>AddMaxSpeed</td><td>0</td><td>115,619</td><td>121,007</td><td>39,269</td><td>52,801</td><td>77,325</td><td>62,429</td><td>468,450</td></tr><tr><td>14</td><td>AddOpeningHours</td><td>51</td><td>34,488</td><td>26,123</td><td>29,432</td><td>47,856</td><td>127,393</td><td>133,161</td><td>398,504</td></tr><tr><td>15</td><td>AddCrossingIsland</td><td>0</td><td>0</td><td>0</td><td>0</td><td>22,209</td><td>202,587</td><td>143,150</td><td>367,946</td></tr><tr><td>16</td><td>AddCrossingType</td><td>0</td><td>10,158</td><td>27,320</td><td>34,726</td><td>47,534</td><td>120,287</td><td>96,071</td><td>336,096</td></tr><tr><td>17</td><td>AddParkingAccess</td><td>0</td><td>0</td><td>18,103</td><td>46,664</td><td>83,911</td><td>111,983</td><td>63,712</td><td>324,373</td></tr><tr><td>18</td><td>AddRoadSmoothness</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>2,545</td><td>277,119</td><td>279,664</td></tr><tr><td>19</td><td>AddBusStopShelter</td><td>0</td><td>23,569</td><td>17,437</td><td>20,509</td><td>27,156</td><td>88,486</td><td>90,175</td><td>267,332</td></tr><tr><td>20</td><td>CheckExistence</td><td>0</td><td>0</td><td>0</td><td>0</td><td>7,377</td><td>164,515</td><td>91,714</td><td>263,606</td></tr><tr><td>21</td><td>AddParkingType</td><td>0</td><td>4,608</td><td>16,574</td><td>25,082</td><td>45,786</td><td>90,480</td><td>77,212</td><td>259,742</td></tr><tr><td>22</td><td>AddRoadName</td><td>34</td><td>40,600</td><td>28,931</td><td>30,785</td><td>51,038</td><td>58,135</td><td>50,076</td><td>259,599</td></tr><tr><td>23</td><td>AddTactilePavingBusStop</td><td>0</td><td>10,678</td><td>16,462</td><td>20,403</td><td>29,128</td><td>103,388</td><td>79,186</td><td>259,245</td></tr><tr><td>24</td><td>AddParkingFee</td><td>0</td><td>0</td><td>11,451</td><td>27,191</td><td>48,507</td><td>85,812</td><td>59,846</td><td>232,807</td></tr><tr><td>25</td><td>AddBenchBackrest</td><td>0</td><td>0</td><td>4,472</td><td>10,399</td><td>22,578</td><td>98,074</td><td>86,518</td><td>222,041</td></tr><tr><td>26</td><td>AddBenchStatusOnBusStop</td><td>0</td><td>0</td><td>0</td><td>0</td><td>18,671</td><td>103,189</td><td>99,630</td><td>221,490</td></tr><tr><td>27</td><td>AddBinStatusOnBusStop</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>88,595</td><td>122,283</td><td>210,878</td></tr><tr><td>28</td><td>AddForestLeafType</td><td>0</td><td>0</td><td>0</td><td>4,096</td><td>23,500</td><td>90,408</td><td>70,462</td><td>188,466</td></tr><tr><td>29</td><td>SidewalkOverlay</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>185,612</td><td>185,612</td></tr><tr><td>30</td><td>AddPathSmoothness</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>1,980</td><td>180,579</td><td>182,559</td></tr><tr><td>31</td><td>AddShoulder</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>13</td><td>174,878</td><td>174,891</td></tr><tr><td>32</td><td>AddBusStopLit</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>74,185</td><td>100,588</td><td>174,773</td></tr><tr><td>33</td><td>AddStepsRamp</td><td>0</td><td>0</td><td>0</td><td>0</td><td>15,498</td><td>84,623</td><td>71,038</td><td>171,159</td></tr><tr><td>34</td><td>WayLitOverlay</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>167,293</td><td>167,293</td></tr><tr><td>35</td><td>AddHandrail</td><td>0</td><td>0</td><td>0</td><td>4,738</td><td>20,936</td><td>68,594</td><td>72,008</td><td>166,276</td></tr><tr><td>36</td><td>AddTrafficSignalsButton</td><td>0</td><td>0</td><td>874</td><td>15,191</td><td>22,466</td><td>63,808</td><td>52,471</td><td>154,810</td></tr><tr><td>37</td><td>AddPowerPolesMaterial</td><td>0</td><td>735</td><td>2,783</td><td>4,036</td><td>10,686</td><td>57,806</td><td>71,993</td><td>148,039</td></tr><tr><td>38</td><td>AddTrafficSignalsSound</td><td>0</td><td>0</td><td>1,014</td><td>14,662</td><td>22,398</td><td>60,651</td><td>46,564</td><td>145,289</td></tr><tr><td>39</td><td>AddTrafficSignalsVibration</td><td>0</td><td>0</td><td>0</td><td>0</td><td>16,099</td><td>64,831</td><td>42,593</td><td>123,523</td></tr><tr><td>40</td><td>AddWheelchairAccessBusiness</td><td>0</td><td>10,024</td><td>9,096</td><td>11,471</td><td>19,141</td><td>35,602</td><td>35,844</td><td>121,178</td></tr><tr><td>41</td><td>AddPlaceName</td><td>0</td><td>304</td><td>0</td><td>15,111</td><td>30,972</td><td>38,547</td><td>33,021</td><td>117,955</td></tr><tr><td>42</td><td>AddStepsIncline</td><td>0</td><td>0</td><td>0</td><td>0</td><td>9,674</td><td>51,711</td><td>47,215</td><td>108,600</td></tr><tr><td>43</td><td>AddTracktype</td><td>0</td><td>0</td><td>418</td><td>7,528</td><td>16,145</td><td>39,860</td><td>33,910</td><td>97,861</td></tr><tr><td>44</td><td>AddStepCount</td><td>0</td><td>0</td><td>0</td><td>0</td><td>7,352</td><td>44,517</td><td>38,512</td><td>90,381</td></tr><tr><td>45</td><td>AddTactilePavingKerb</td><td>0</td><td>0</td><td>0</td><td>0</td><td>4,542</td><td>45,659</td><td>39,437</td><td>89,638</td></tr><tr><td>46</td><td>AddKerbHeight</td><td>0</td><td>0</td><td>0</td><td>0</td><td>3,767</td><td>43,395</td><td>38,621</td><td>85,783</td></tr><tr><td>47</td><td>AddMaxHeight</td><td>0</td><td>0</td><td>1,868</td><td>5,509</td><td>8,034</td><td>37,034</td><td>31,797</td><td>84,242</td></tr><tr><td>48</td><td>AddPlaygroundAccess</td><td>0</td><td>0</td><td>2,139</td><td>7,281</td><td>11,544</td><td>37,465</td><td>25,496</td><td>83,925</td></tr><tr><td>49</td><td>AddSidewalkSurface</td><td>0</td><td>0</td><td>0</td><td>0</td><td>3</td><td>0</td><td>81,063</td><td>81,066</td></tr><tr><td>50</td><td>AddBikeParkingCover</td><td>0</td><td>1,884</td><td>3,930</td><td>5,508</td><td>8,298</td><td>26,162</td><td>25,078</td><td>70,860</td></tr><tr><td>51</td><td>MarkCompletedHighwayConstruction</td><td>0</td><td>0</td><td>2,055</td><td>3,857</td><td>7,603</td><td>31,181</td><td>25,450</td><td>70,146</td></tr><tr><td>52</td><td>StreetParkingOverlay</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>63,893</td><td>63,893</td></tr><tr><td>53</td><td>AddCyclewaySegregation</td><td>0</td><td>0</td><td>1,899</td><td>7,571</td><td>8,341</td><td>24,273</td><td>21,486</td><td>63,570</td></tr><tr><td>54</td><td>AddBikeParkingCapacity</td><td>0</td><td>2,551</td><td>3,423</td><td>3,486</td><td>7,022</td><td>22,793</td><td>18,069</td><td>57,344</td></tr><tr><td>55</td><td>AddPitchSurface</td><td>0</td><td>0</td><td>0</td><td>0</td><td>58</td><td>34,852</td><td>21,576</td><td>56,486</td></tr><tr><td>56</td><td>AddFootwayPartSurface</td><td>0</td><td>0</td><td>0</td><td>1,830</td><td>9,740</td><td>25,979</td><td>18,751</td><td>56,300</td></tr><tr><td>57</td><td>AddCyclewayPartSurface</td><td>0</td><td>0</td><td>0</td><td>1,988</td><td>10,652</td><td>25,488</td><td>16,385</td><td>54,513</td></tr><tr><td>58</td><td>AddRailwayCrossingBarrier</td><td>0</td><td>0</td><td>7,177</td><td>9,169</td><td>5,407</td><td>12,883</td><td>19,340</td><td>53,976</td></tr><tr><td>59</td><td>AddProhibitedForPedestrians</td><td>0</td><td>0</td><td>0</td><td>3,520</td><td>7,099</td><td>24,463</td><td>17,704</td><td>52,786</td></tr><tr><td>60</td><td>AddFireHydrantPosition</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>60</td><td>50,225</td><td>50,285</td></tr><tr><td>61</td><td>AddPitchLit</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>25,425</td><td>24,157</td><td>49,582</td></tr><tr><td>62</td><td>AddMaxWeight</td><td>0</td><td>0</td><td>0</td><td>929</td><td>6,576</td><td>20,605</td><td>18,447</td><td>46,557</td></tr><tr><td>63</td><td>AddBikeParkingType</td><td>0</td><td>0</td><td>2,354</td><td>4,186</td><td>5,855</td><td>17,720</td><td>15,840</td><td>45,955</td></tr><tr><td>64</td><td>AddCrossing</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>16,581</td><td>28,314</td><td>44,895</td></tr><tr><td>65</td><td>AddBollardType</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>19,458</td><td>22,269</td><td>41,727</td></tr><tr><td>66</td><td>AddRoadWidth</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>40,795</td><td>40,795</td></tr><tr><td>67</td><td>AddPostboxCollectionTimes</td><td>0</td><td>0</td><td>2,076</td><td>3,113</td><td>5,489</td><td>15,853</td><td>8,445</td><td>34,976</td></tr><tr><td>68</td><td>AddFireHydrantType</td><td>0</td><td>322</td><td>472</td><td>1,394</td><td>2,345</td><td>9,707</td><td>15,455</td><td>29,695</td></tr><tr><td>69</td><td>AddRecyclingContainerMaterials</td><td>0</td><td>0</td><td>0</td><td>15</td><td>5,130</td><td>12,000</td><td>11,487</td><td>28,632</td></tr><tr><td>70</td><td>AddBridgeStructure</td><td>0</td><td>0</td><td>528</td><td>1,204</td><td>1,945</td><td>11,255</td><td>13,506</td><td>28,438</td></tr><tr><td>71</td><td>AddVegetarian</td><td>0</td><td>341</td><td>2,575</td><td>3,323</td><td>4,213</td><td>6,796</td><td>8,765</td><td>26,013</td></tr><tr><td>72</td><td>AddSeating</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>22,262</td><td>22,262</td></tr><tr><td>73</td><td>AddOneway</td><td>0</td><td>0</td><td>2,424</td><td>2,639</td><td>3,385</td><td>774</td><td>11,665</td><td>20,887</td></tr><tr><td>74</td><td>AddPicnicTableCover</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>4,681</td><td>16,194</td><td>20,875</td></tr><tr><td>75</td><td>AddRecyclingType</td><td>0</td><td>4,149</td><td>4,789</td><td>2,772</td><td>1,878</td><td>2,921</td><td>2,157</td><td>18,666</td></tr><tr><td>76</td><td>AddToiletsFee</td><td>0</td><td>466</td><td>459</td><td>725</td><td>1,236</td><td>7,167</td><td>8,151</td><td>18,204</td></tr><tr><td>77</td><td>MarkCompletedBuildingConstruction</td><td>0</td><td>0</td><td>215</td><td>867</td><td>1,448</td><td>7,401</td><td>7,747</td><td>17,678</td></tr><tr><td>78</td><td>AddFireHydrantDiameter</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>2,811</td><td>14,254</td><td>17,065</td></tr><tr><td>79</td><td>DetailRoadSurface</td><td>0</td><td>0</td><td>0</td><td>0</td><td>16,690</td><td>93</td><td>0</td><td>16,783</td></tr><tr><td>80</td><td>AddSport</td><td>0</td><td>2,379</td><td>1,615</td><td>1,795</td><td>2,830</td><td>4,852</td><td>3,174</td><td>16,645</td></tr><tr><td>81</td><td>AddBusStopName</td><td>0</td><td>0</td><td>545</td><td>1,623</td><td>3,624</td><td>4,711</td><td>2,780</td><td>13,283</td></tr><tr><td>82</td><td>AddBoardType</td><td>0</td><td>0</td><td>0</td><td>0</td><td>968</td><td>6,338</td><td>5,890</td><td>13,196</td></tr><tr><td>83</td><td>AddWheelchairAccessToilets</td><td>0</td><td>0</td><td>0</td><td>408</td><td>1,154</td><td>5,781</td><td>5,392</td><td>12,735</td></tr><tr><td>84</td><td>AddBicycleBarrierType</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>1,663</td><td>9,698</td><td>11,361</td></tr><tr><td>85</td><td>AddVegan</td><td>0</td><td>135</td><td>911</td><td>1,053</td><td>1,453</td><td>2,890</td><td>4,240</td><td>10,682</td></tr><tr><td>86</td><td>AddAcceptsCards</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>10,610</td><td>10,610</td></tr><tr><td>87</td><td>CheckShopType</td><td>0</td><td>0</td><td>0</td><td>0</td><td>160</td><td>4,066</td><td>5,049</td><td>9,275</td></tr><tr><td>88</td><td>AddCyclewayWidth</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>9,057</td><td>9,057</td></tr><tr><td>89</td><td>AddEntrance</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>8,856</td><td>8,856</td></tr><tr><td>90</td><td>AddWheelchairAccessPublicTransport</td><td>0</td><td>0</td><td>0</td><td>440</td><td>966</td><td>3,971</td><td>3,381</td><td>8,758</td></tr><tr><td>91</td><td>AddAirCompressor</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>7,755</td><td>7,755</td></tr><tr><td>92</td><td>AddKosher</td><td>0</td><td>0</td><td>0</td><td>0</td><td>4</td><td>4,121</td><td>3,492</td><td>7,617</td></tr><tr><td>93</td><td>AddHalal</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>3,240</td><td>3,895</td><td>7,135</td></tr><tr><td>94</td><td>DetermineRecyclingGlass</td><td>0</td><td>0</td><td>0</td><td>36</td><td>1,373</td><td>2,982</td><td>2,524</td><td>6,915</td></tr><tr><td>95</td><td>AddGeneralFee</td><td>0</td><td>0</td><td>0</td><td>133</td><td>515</td><td>2,487</td><td>3,670</td><td>6,805</td></tr><tr><td>96</td><td>AddressOverlay</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>6,207</td><td>6,207</td></tr><tr><td>97</td><td>AddCarWashType</td><td>0</td><td>38</td><td>207</td><td>244</td><td>434</td><td>2,392</td><td>2,806</td><td>6,121</td></tr><tr><td>98</td><td>AddToiletAvailability</td><td>0</td><td>93</td><td>315</td><td>505</td><td>576</td><td>2,200</td><td>2,416</td><td>6,105</td></tr><tr><td>99</td><td>AddSmoking</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>5,891</td><td>5,891</td></tr><tr><td>100</td><td>AddMotorcycleParkingCover</td><td>0</td><td>0</td><td>5</td><td>302</td><td>611</td><td>2,410</td><td>2,287</td><td>5,615</td></tr></tbody></table>

I was surprised that over a million house numbers were added with StreetComplete, since I hardly see any missing house numbers in Germany, but I guess this information is missing a lot in other parts of the world. Overall I think this table is quite interesting.

If you have any other questions about the usage of StreetComplete, ideas about plots or feedback in general, you can open an issue in the [repo](https://github.com/piebro/openstreetmap-statistics) or comment here.

**Comment by HolgerJeromin** [comment]
> There are on average over 1M edits each month in the last two years after an interesting spike at the end of 2020. I wonder why the growth is so sudden.

The growth is matching release of v26:
https://github.com/streetcomplete/StreetComplete/releases/tag/v26.0

So after that all quests are immediately visible in the app. More and quicker quests => more answered quest.

**Comment by matkoniecz** [comment]
https://github.com/streetcomplete/StreetComplete/discussions/2671#discussioncomment-787133 has focus on this jump with releases marked

**Comment by FloEdelmann** [comment]
@piebro, maybe it's possible to automatically fetch the release dates of all non-alpha/beta StreetComplete versions via the GitHub API and display them as vertical lines in the graph?

See https://docs.github.com/en/rest/releases/releases

**Comment by piebro** [comment]
> The growth is matching release of v26: https://github.com/streetcomplete/StreetComplete/releases/tag/v26.0
> 
> So after that all quests are immediately visible in the app. More and quicker quests => more answered quest.

Oh yeah, that's true, but more quest is probably not the only explanation because looking [here](https://piebro.github.io/openstreetmap-statistics/#465b)
![contributors_that_use_streetcomplete_per_month](https://user-images.githubusercontent.com/17904272/203826339-ad207bed-ff9d-46cb-b4bb-8f78fe94e851.png)
the number of monthly contributors is raising too. Its probably because the v26 release was also more popular.

> @piebro, maybe it's possible to automatically fetch the release dates of all non-alpha/beta StreetComplete versions via the GitHub API and display them as vertical lines in the graph?

Oh yeah, that's a great idea, I will look into that, thanks!

**Comment by matkoniecz** [comment]
> I was surprised that over a million house numbers were added with StreetComplete, since I hardly see any missing house numbers in Germany

That is not a number of added house numbers - that is count how many times this quest was answered. It may seem pedantic, but I encountered cases where groups of garages were mapped as `building=house` or `building=retail` where asking shopkeeper revealed that house number is not assigned there ( https://www.openstreetmap.org/way/1116288092 https://www.openstreetmap.org/node/10209776391 )

See also https://www.reddit.com/r/openstreetmap/comments/ywgzna/this_is_what_street_complete_looks_like_in/

![screen02](https://user-images.githubusercontent.com/899988/203835239-bd507489-bd3f-4843-a40b-3729921424e7.png)

**Comment by westnordost** [comment]
Whoa, awesome!
